src/runtime_src/core/tools/common/XBUtilities.h

```diff
@@ -22,16 +23,37 @@
 #include "core/common/device.h"
 #include "core/common/query_requests.h"
 
+#include <chrono>
+#include <iostream>
 #include <string>
 #include <memory>
 #include <map>
-#include <iostream>
 #include <vector>
-#include <boost/property_tree/ptree.hpp>
-#include <boost/program_options.hpp>
+
```

Comment:
While you are here, could you contribute to our community and sort the `#include`? :-)